GDJS/Runtime/runtimeobject.ts

```diff
@@ -388,6 +388,23 @@ namespace gdjs {
       return this.getY();
     }
 
+    /**
+     * Rotate a point in the space
+     * @return The X and Y position of the rotated point.
+     */
+    rotatePoint(
+      point: { x: float; y: float },
+      origin: { x: float; y: float },
+      angle: float
+    ): { x: float; y: float } {
+      angle *= Math.PI / 180;
+      const pointX = point.x - origin.x;
+      const pointY = point.y - origin.y;
+      const x = pointX * Math.cos(angle) + pointY * Math.sin(angle) + origin.x;
+      const y = -pointX * Math.sin(angle) + pointY * Math.cos(angle) + origin.y;
+      return { x: Math.round(x), y: Math.round(y) };
+    }
```

Comment:
Could you use `getPointX`/`getPointY` (only for sprite, but that's fine you can do a `if (obj instanceof gdjs.SpriteRuntimeObject) { ... }`? Note that for it to be efficient, we can add a `getPointPosition` that returns the array (instead of calling `getPointX` and then `getPointY`, which is inefficient).